bin/civi-download-tools

```diff
@@ -21,9 +21,6 @@ LOCKTIMEOUT=90
 CIVIXURL="https://download.civicrm.org/civix/civix.phar-2017-08-02-2489efea"
 CVURL="https://download.civicrm.org/cv/cv.phar-2017-07-27-89ba2508"
 HUB_VERSION="2.2.9"
-DRUSH8URL=http://files.drush.org/drush.phar
-PHPUNITURL=https://phar.phpunit.de/phpunit-4.8.21.phar
-WPCLIURL=https://github.com/wp-cli/wp-cli/releases/download/v1.1.0/wp-cli-1.1.0.phar
```

Comment:
As @kcristiano said:

> One comment is that we have had issues pulling wp-cli via composer before due to conflicts with dependencies.

Originally, `buildkit` downloads were almost entirely handled via `composer`, but that style of distribution basically requires that *all CLI tools* use the same versions of their libraries. The `composer` global style has the same problem. PHAR-based distribution solves this. (I have a fairly strong suspicion that this is why most PHP-CLI tools recommend their PHAR-based builds.)

It would be prettier if these requirements were listed in some kind of JSON file. You could potentially write a small PHP script and JSON file which handles this in a nicer fashion. If you really wanted it in`composer.json`, it be a [plugin](https://getcomposer.org/doc/articles/plugins.md). Ex:

```json
  "require": {
    "composer/phar-plugin" : "^1.0"
  },
  "extra": {
    "phar": {
      "bin/phpunit4": "https://phar.phpunit.de/phpunit-4.8.21.phar",
      "bin/wp": "https://github.com/wp-cli/wp-cli/releases/download/v1.1.0/wp-cli-1.1.0.phar"
    }
  }
```

(Update: These look interesting but seem to require PHP 5.6: https://github.com/tommy-muehle/composer-tool-installer-plugin and https://github.com/tommy-muehle/tooly-composer-script )